containers/README.md

```diff
@@ -12,3 +12,48 @@ Table of Contents
  * [Issues to be Addressed](#issues)
 
 For diagrams outlining the architecture, please see the diagrams in the `docs/` folder.
+
+## Quick Start Guide
+
+This quickstart guide requires that the following are installed already:
+
+ * [Ansible](http://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+ * [Minishift](https://docs.openshift.org/latest/minishift/getting-started/installing.html)
+
+System Requirements:
+
+ * 10GB RAM free
+
+### Start Minishift
+
+The minishift setup requires some customization due to resource requirements. Run the following to start minishift:
+
+    minishift start --memory 10GB --cpus 4 --iso-url centos
+
+A playbook is also provided:
+
+    ansible-playbook tools/minishift-start.yml
```

Comment:
from the instructions its not clear if I should use that instead of minishift start above?